client/flutter/lib/l10n/intl_es.arb

```diff
@@ -1,23 +1,26 @@
 {
-    "worldHealthOrganization": "",
-    "learnMore": "",
-    "protectYourself": "",
-    "checkYourHealth": "",
-    "localMaps": "",
-    "shareTheApp": "",
-    "healthCheck": "",
-    "feelingDistressed": "",
-    "washHands": "",
-    "cougningAndSneezing": "",
-    "throwAwayTissue": "",
-    "washHandsFrequently": "",
-    "socialDistancing": "",
-    "seekMedicalCare": "",
-    "infoForEveryone":"",
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
+    "worldHealthOrganization": "Organización Mundial de la Salud",
+    "learnMore": "Aprenda más",
+    "protectYourself": "Protéjase",
+    "checkYourHealth": "Consulte su estado de salud",
+    "localMaps": "Mapas Locales",
+    "shareTheApp": "Compartir la aplicación",
+    "healthCheck": "Consulta de salud",
+    "feelingDistressed": "Se siente angustiado/a?",
+    "washHands": "Lávese las manos a menudo con agua y jabón",
+    "cougningAndSneezing": "Cuando tosa o estornude, tápese la boca y la nariz con un pañuelo, o con su codo",
+    "throwAwayTissue": "Eche a la papelera inmediatamente cualquier pañuela después de su uso",
+    "washHandsFrequently": "Lávese las manos frecuentemente",
+    "socialDistancing": "Evite cualquier contacto con personas, y mantenga una distancia de separación prudencial",
+    "seekMedicalCare": "Busque asistencia sanitaria si tiene fiebre, tos, o dificultad al respirar",
+    "infoForEveryone":"Información para Todos/as",
+    "infoForParents":"Información para Padres y Madres",
+    "childStress":"Los niños responden al estrés de forma muy diferente, siendo más ansiosos, agitates, mojando la cama...\nResponde a los niños de forma que se sientan protegidos, escucha sus preocupaciones, y dales un extra de amor y atención",
+    "childLove":"Los niños necesitan la atención y el amor de los adultos en los momentos más difíciles. Dedícales un extra de tiempo y atención.\nNo olvides escucharles, hablarles educadamente y de forma tranquilizadora.\nSi es posible, dales la oportunidad de jugar y poder relajarse.",
```

Comment:
"Dedícales" is too personal ... but I'm not sure of which should be the 'tone' of this app.
Maybe something to take into consideration and use a more formal one? "Dedíqueles".
This would be for almost all of these translations

Reply:
I see along this translation that several tones are used. 
**Formal & Informal**
Maybe we should consider go for one of them only for allowing consistency across the app,